README.md

```diff
@@ -276,6 +276,7 @@ TOC is generated by [markdown-toc](https://github.com/jonschlinkert/markdown-toc
 - [Vexu/bog](https://github.com/Vexu/bog) : Small, strongly typed, embeddable language.
 - [squeek502/zua](https://github.com/squeek502/zua) : An implementation of Lua 5.1 in Zig, for learning purposes.
 - [buzz](https://github.com/buzz-language/buzz): A small/lightweight statically typed scripting language.
+- [dockerc](https://github.com/NilsIrl/dockerc): container image to single executable compiler.
```

Comment:
I prefer to use add this repo under this section.

https://github.com/NilsIrl/dockerc/blob/trunk/src/dockerc.zig#L37

It shell out to `umoci` to do the heavy task, I prefer to add library mainly written in Zig.

`Others` may be suitable.

Reply:
okay, adding to others again 😆